packages/web-app-admin-settings/src/views/Users.vue

```diff
@@ -376,31 +368,26 @@ export default defineComponent({
         })
       }
     },
-    async editUser(editUser) {
+    async editUser({ user, editUser }) {
       try {
-        const user = this.users.find((user) => user.id === editUser.id)
-        const actualUser = {
-          ...(user && user),
-          passwordProfile: { password: '' }
-        }
-
         const graphEditUserRawObjectExtractor = (user) => {
-          return omit(user, ['drive', 'role'])
+          return omit(user, ['drive', 'appRoleAssignments', 'memberOf'])
         }
 
+        console.log((graphEditUserRawObjectExtractor(editUser)))
```

Comment:
remove dev leftovers please :-)

Reply:
Yes, something for tomorrow we try to figure out why everything is messed up with the object equals stuff